[info_terminal] Fails to install

```
error: package directory 'src/approach_person_of_interest' does not exist
CMake Error at aaf_deployment/info_terminal/approach_person_of_interest/catkin_generated/safe_execute_install.cmake:4 (message):
  
  execute_process(/home/cdondrup/ros_ws/build/aaf_deployment/info_terminal/approach_person_of_interest/catkin_generated/python_distutils_install.sh)
  returned error code
Call Stack (most recent call first):
  aaf_deployment/info_terminal/approach_person_of_interest/cmake_install.cmake:36 (INCLUDE)
  cmake_install.cmake:141 (INCLUDE)
```

--- a/info_terminal/approach_person_of_interest/setup.py
+++ b/info_terminal/approach_person_of_interest/setup.py
@@ -5,7 +5,7 @@ from catkin_pkg.python_setup import generate_distutils_setup
 
 # fetch values from package.xml
 setup_args = generate_distutils_setup(
-    packages=['approach_person_of_interest'],
+    packages=[],
     package_dir={'': 'src'})
 
 setup(**setup_args)

